assembly/src/parsers/serde/mod.rs

```diff
@@ -249,4 +249,5 @@ pub enum OpCode {
     CallLocal = 215,
     CallImported = 216,
     SysCall = 217,
+    Incr = 218,
```

Comment:
I would keep the order the same as in other places (i.e., put this after `Inv`). I know this will update mode values here, but I think that's OK.